WordPressPCL/WordPressPCL.xml

```diff
@@ -1044,6 +1043,11 @@
             Comment is spam
             </summary>
         </member>
+        <member name="F:WordPressPCL.Models.CommentStatus.Trash">
+            <summary>
+            Comment is is trash
```

Comment:
Probably you meant "Comment is in trash."

Reply:
Fixed, thanks